week-2/D-predicates/exercise.js

```diff
@@ -7,12 +7,12 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+ return number < 0 ;
 }
 
 // Finish the predicate function to test if the passed number is between 0 and 10
 function isBetweenZeroAnd10(number) {
-
+ return 0 < number < 10;
```

Comment:
This won't work. One difference between javascript and common math is that in javascript, you can only compare two things at a time. So, if the number was, say, 11, the computer is actually doing:
`(0 < 11) < 10`, which then gets evaluated to `true < 10` Since boolean value true is being compared as if it's a number, it gets converted into `1 < 10` which results in true. This is not what you expected.

How can you phrase this such that you are doing comparisons between two numbers at a time?